include/cantera/kinetics/InterfaceRate.h

```diff
@@ -93,6 +111,95 @@ class CoverageBase
             m_ecov += m_ec[item.first] * shared_data.coverages[item.second];
             m_mcov += m_mc[item.first] * shared_data.logCoverages[item.second];
         }
+
+        // Update change in electrical potential energy
+        if (m_chargeTransfer) {
+            m_deltaPotential_RT = 0.;
+            for (const auto& ch : m_netCharges) {
+                m_deltaPotential_RT +=
+                    shared_data.electricPotentials[ch.first] * ch.second;
+            }
+            m_deltaPotential_RT /= GasConstant * shared_data.temperature;
+        }
+
+        // Update quantities used for exchange current density formulation
+        if (m_exchangeCurrentDensityFormulation) {
+            m_deltaGibbs0_RT = 0.;
+            m_prodStandardConcentrations = 1.;
+            for (const auto& item : m_stoichCoeffs) {
+                m_deltaGibbs0_RT +=
+                    shared_data.standardChemPotentials[item.first] * item.second;
+                if (item.second > 0.) {
+                    m_prodStandardConcentrations *=
+                        shared_data.standardConcentrations[item.first];
+                }
+            }
+            m_deltaGibbs0_RT /= GasConstant * shared_data.temperature;
+        }
+    }
+
+    //! Calculate modifications for the forward reaction rate for interfacial charge
+    //! transfer reactions.
+    /*!
+     *  For reactions that transfer charge across a potential difference, the
+     *  activation energies are modified by the potential difference. The correction
+     *  factor is based on the net electric potential energy change
+     *  \f[
+     *   deltaElectricEnergy = sum_i ( pot_i nu_ij)
```

Comment:
The formatting of this equation could use some work (sorry if this is just being moved from elsewhere)

Reply:
Yup. Copy-pasta ...